### PR TITLE
June 14 Release Notes

### PR DESCRIPTION
**Pull request by lbadenhop**
Thank you for your contribution to the Amplify-Central repo.

## Describe the changes

Enter a brief description of your changes to communicate to the reviewers what you changed and why.

## Deploy preview link

Please add the deploy preview link to the **specific page** that you've changed.

## Checklist for contributors

Before submitting this PR:

* Verify that all status checks have passed
* For more information on the Markdown linter rules, see [DavidAnson/markdownlint](https://github.com/DavidAnson/markdownlint)
* For first-time contributors, ensure that you have signed the [Axway CLA](https://cla.axway.com/)

### CODE REVIEW

**Comment by docsbuilder** [comment]
### <span aria-hidden="true">✅</span> Deploy preview is ready!

 |  Name | Link |
 |---------------------------------|------------------------|
 |<span aria-hidden="true">🔨</span> Latest commit | 5d6956b |
 |<span aria-hidden="true">😎</span> Deploy preview | http://PR-466.amplify-central.opendocs-builder.pcloud.axway.int |

File: content/en/docs/amplify_relnotes/20240614_amplify.md [hunk 1]
```diff
@@ -0,0 +1,189 @@
...
+* Go to [https://repository.axway.com/catalog?q=agents](https://repository.axway.com/catalog?q=agents).
+
+### Axway Agent SDK
+
+The following enhancements and bug fixes are available in this release.
+
+#### Enhancements for Agent SDK
+
+* **Region variable setting** - An enhancencement was made for all agents to support a "CENTRAL_REGION" variable which can be set to US, EU, or APAC. This would enable the agent to automatically determine the URL to use in each region to connect to Amplify.
+* **Dynamic Client Registration settings** - An enhancement has been made to enable the configuration of additional Dynamic Client Regstration protocol settings (e.g. to support Oracle Access Maanger).
```
Comment (jcollins-axway) [inline content/en/docs/amplify_relnotes/20240614_amplify.md:57]: I realize these changes are in the SDK, but does it make sense to list it here and under v7?

File: content/en/docs/amplify_relnotes/20240614_amplify.md [hunk 2]
```diff
@@ -0,0 +1,189 @@
...
+    * AGENTFEATURES_IDP_REQUESTHEADERS - Additional request headers to send in the request when registering a new client.
+    * AGENTFEATURES_IDP_QUERYPRAMS - Additional query parameters to send in the request when registering a new client.
+    * AGENTFEATURES_IDP_AUTH_REQUESTHEADERS - Additional request headers to send in the request for getting a token.
+    * AGENTFEATURES_IDP_AUTH_QUERYPARAMS - Additional query parameters to send in the request for getting a token.
+
+##### Bug fixes for Axway API Management Gateway Agent
+
+| Case ID     | Internal ID  | Description                                       |
+|---------    |--------------|---------------------------------------------------|
+| 01570727    | APIGOV-28012 | **Issue**: In some deployment situations, the Traceability Agent could not be deployed with a Kubernetes StatefulSet. <br />**Resolution**: The StatefulSet deployment is optional. |
```
Comment (jcollins-axway) [inline content/en/docs/amplify_relnotes/20240614_amplify.md:88]: The issue is more "The Traceability Agent only supported deployment as a DaemonSet..."

Comment (robertengaxway) [reply content/en/docs/amplify_relnotes/20240614_amplify.md:88]: Made edits after discussion with Jason

File: content/en/docs/amplify_relnotes/20240614_amplify.md (hunk 1 above)
Comment (robertengaxway) [reply content/en/docs/amplify_relnotes/20240614_amplify.md]: Removed the duplication from the V7 Agent section.   Added a line in the Agent SDK to indicate this is supported for V7 and Istio Agents only.

**Review by robertengaxway — APPROVED** [review]
I approve

**Review by jcollins-axway — APPROVED** [review]

File: content/en/docs/amplify_relnotes/20240614_amplify.md [hunk 3]
```diff
@@ -0,0 +1,198 @@
...
+
+### Axway Central CLI
+
+The latest version of the [Axway Central CLI is v2.28.0 on NPM](https://www.npmjs.com/package/@axway/axway-central-cli/v/2.28.0). There are no enhancements or bug fixes in this release.
+
+### Service Registry
+
+The following enhancements are available in this release.
+
+* **Mock Service support** - An enhancencement has been made to integrate a Mock service to simulate an API endpoint early in the API lifecycle based upon its API Specification. The Mocked endpoints can be created from the Service Registry, the API Service details, the API Service wizard from the Environments details page, Services tab on the Environment details page.
```
Comment (georgiana-caras) [inline content/en/docs/amplify_relnotes/20240614_amplify.md:122]: Through the embedded Mocking Service feature, providers now have the ability to create a mock endpoint early in theAPI lifecycle based upon its API Specification.

**Review by georgiana-caras — CHANGES_REQUESTED** [review]

**Review by georgiana-caras — APPROVED** [review]

File: content/en/docs/amplify_relnotes/20240614_amplify.md (hunk 3 above)
Comment (robertengaxway) [reply content/en/docs/amplify_relnotes/20240614_amplify.md]: made that edit

**Pull request merged**